### PR TITLE
VSCode fixes

### PR DESCRIPTION
**Pull request by PaulRitter**
vscode shows errors/warning, i fixed alot of em.

basically all remaining i don't understand how to fix

### CODE REVIEW

**Comment by PaulRitter** [comment]
oh boy 4 conflicting files

**Comment by boy2mantwicethefam** [comment]
Oh fuck my sides

**Comment by PaulRitter** [comment]
eat my dust grandpa

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -511,10 +511,10 @@
 
 /*
  * Syndie stealthy smokebombs!
- */
- /obj/item/toy/snappop/smokebomb
- 	origin_tech = Tc_COMBAT + "=1;" + Tc_SYNDICATE + "=1"
+*/
+/obj/item/toy/snappop/smokebomb
 	flags = FPRINT | NO_THROW_MSG
+	origin_tech = Tc_COMBAT + "=1;" + Tc_SYNDICATE + "=1";
```
Comment (PJB3005) [inline code/game/objects/items/toys.dm]: That semicolon is not needed. The problem on this line was the horrendous indentation.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (PaulRitter) [reply code/game/objects/items/toys.dm]: ah ok, i was wondering why it was complaining there

**Pull request merged**